assets/video-section.css

```diff
@@ -4,6 +4,10 @@
   padding-bottom: calc(var(--ratio-percent) - var(--media-border-width));
 }
 
+.scroll-trigger:where(.gradient.video-section__media) {
+  background: transparent;
+}
```

Comment:
Video section with animation and without. 

<details>
<summary>Comparison screenshot</summary>

![](https://screenshot.click/17-56-knsm2-yzsec.png)
</details>

Reply:
Is this meant to document that there are no changes between before and after?